src/models/file.ts

```diff
@@ -75,16 +75,7 @@ export default class File extends RestfulModel {
         path: `/files/${this.id}/download`,
         downloadRequest: true,
       })
-      .then(response => {
-        let filename;
```

Comment:
This is safe to delete because we never use the filename in this block nor do we return it.